src/components/MenuPanel/WalletManager/Main/Main.js

```diff
@@ -18,7 +18,6 @@ type Props = {|
   +address: string,
   +isActive: boolean,
   +isReadOnly: boolean,
-  +isSimplified: boolean,
```

Comment:
please remove this prop from passing to this component

Reply:
Fixed